subsys/spm/spm.c

```diff
@@ -567,3 +568,60 @@ void spm_config(void)
 	spm_config_sram();
 	spm_config_peripherals();
 }
+
+struct spm_uart_config {
+	const struct device *uart;
+	NRF_UARTE_Type *regs;
+};
+
+#ifdef CONFIG_UART_0_NRF_UARTE
+
+static bool spm_periph_reconf(uint8_t id, bool secure)
```

Comment:
It is not clear what this function does, can you provide a @brief with a oneliner explaining it?